licensing/src/main/java/io/crate/license/LicenseService.java

```diff
@@ -130,7 +137,15 @@ public boolean hasValidLicense() {
         if (enterpriseEnabled == false) {
             return true;
         }
-        return !isLicenseExpired(currentLicense());
+        // We consider an instance that is bound to loopback as a development instance and by-pass the license expiration.
+        // This makes it easier for us to run our tests.
+        return boundToLocalhost() || !isLicenseExpired(currentLicense());
```

Comment:
also agree that we could store the `boundToLocalhost()` result into a instance var.

Reply:
Are you sure about that? 
Can't nodes be added and leave in the component lifecycle?

I was referring to the predicate definition, not the evaluation.

Reply:
If nodes are added or leave it doesn't influence the bound address. That is static. Even if the transport lifecycle component would be stopped & started again, it won't change the outcome of the evaluation as the bind-host settings are not runtime settings.